packages/dds/tree/src/tableSchema.ts

```diff
@@ -852,6 +852,19 @@ export namespace System_TableSchema {
 /**
  * Contains types and factories for creating schema to represent dynamic tabular data.
  *
+ * @remarks
+ *
+ * Tables created using these APIs are...
+ *
+ * - sparse, meaning that they only store the cells that are actually present in the table.
```

Comment:
This seems a little self referential. A dense table also only stores cells in the table.

Also it's not really accurate since this table can store cells that are not in the table, like ones that are part of a removed column.

maybe:

```suggestion
 * - sparse, meaning that any given could be omitted from the table.
```

Reply:
Yeah, this is not what I meant. Should have scrutinized Copilot's suggestion better. I'll update this.